api/query/cache/service/main.go

```diff
@@ -158,12 +159,25 @@ func main() {
 	var netClient = &http.Client{
 		Timeout: time.Second * 5,
 	}
+
+	// Initializes Logger.
+	var gclogClient *gclog.Client
+
+	gclogClient, err = gclog.NewClient(context.Background(), *projectID)
```

Comment:
Would this work?
```suggestion
	gclogClient, err := gclog.NewClient(context.Background(), *projectID)
```

Reply:
Yup it will work. I just thought that declaring `gclogClient` would make it more clear on what is being deferred.